Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALCalibExtractor.h

```diff
@@ -40,12 +40,14 @@ class EMCALCalibExtractor
   using boostHisto = boost::histogram::histogram<std::tuple<boost::histogram::axis::regular<double, boost::use_default, boost::use_default, boost::use_default>, boost::histogram::axis::integer<>>, boost::histogram::unlimited_storage<std::allocator<char>>>;
 
  public:
-  EMCALCalibExtractor() = default;
+  EMCALCalibExtractor()
+  {
+    LOG(info) << "initialized EMCALCalibExtractor";
+    mGeometry = o2::emcal::Geometry::GetInstanceFromRunNumber(300000);
```

Comment:
Ideally the init should check if the singleton already exist and only if not initialize it with a run number. This will also solve the run number issue as we will initalize the geometry in the EMCALChannelCalibrator (Device) which at some point will hopefully have access to the run number.